pyproject.toml

```diff
@@ -203,6 +203,8 @@ include-package-data = true
 include = [
 	"setuptools*",
 	"pkg_resources*",
+	# TODO: Include distutils stubs with package once we're confident in them
+	# "typings/distutils-stubs",
```

Comment:
We don't need to ship anything as long as typeshed provides stubs for `setuptools/_distutils`. We can synchronize once ready.
As I mentioned in the issue, there's also always the option to ship it as a different package 9in which case it could live completely separately from setuptools and pypa/distutils)